src/main/java/com/google/security/zynamics/binnavi/API/debug/raw/RegisterValues.java

```diff
@@ -43,14 +44,15 @@
   // / @endcond
   public RegisterValues(
       final com.google.security.zynamics.binnavi.debug.models.targetinformation.RegisterValues registerValues) {
-    final List<ThreadRegisterValues> values = new ArrayList<ThreadRegisterValues>();
-
-    for (final ThreadRegisters threadRegisterValues : registerValues) {
+    
+    final List<ThreadRegisterValues> values = new ArrayList<ThreadRegisterValues>();		
+ 		 
+    for (final ThreadRegisters threadRegisterValues : registerValues) {	
       values.add(new ThreadRegisterValues(threadRegisterValues));
     }
-
-    this.values = values;
-  }
+		
+    this.values = values;		
```

Comment:
please remove the spaces. http://clang.llvm.org/docs/ClangFormat.html with Google Formatting.